From daeb25eb027cef2847d1f508d944965fc526b5e6 Mon Sep 17 00:00:00 2001
From: Marko Hammersen <marko.hammersen@gmail.com>
Date: Fri, 28 Nov 2025 15:34:00 +0100
Subject: [PATCH] yocto assignment 7

---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b08c49..c4f2e2e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,5 @@
 
-SUBDIRS =  misc-progs misc-modules \
-           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-	   pci usb\
-	   sbull snull short
+SUBDIRS =  misc-modules scull
 
 all: subdirs
 
